src/DependencyInjection/Configuration.php

```diff
@@ -55,6 +70,12 @@ public function getConfigTreeBuilder()
                         ->thenInvalid('%s')
                     ->end()
                 ->end()
+                ->booleanNode('image_cache')
```

Comment:
I was thinking about adding a sub-namespace for images. Will be useful for @ausi's PR too…